cpp/tests/join/join_tests.cpp

```diff
@@ -1659,4 +1636,14 @@ TEST_F(JoinTest, FullJoinWithStructsAndNulls)
   CUDF_TEST_EXPECT_TABLES_EQUIVALENT(*sorted_gold, *sorted_result);
 }
 
+TEST_F(JoinTest, LargeJoinResult)
```

Comment:
This test will _probably_ fail on CI depending on how much GPU memory is available. Open to suggestions for testing (or not testing) here.

Reply:
Not testing